dart/dynamics/InverseKinematics.h

```diff
@@ -478,7 +559,7 @@ class InverseKinematics : public common::Subject
 
     /// Constructor
     explicit JacobianTranspose(InverseKinematics* _ik,
-                               double _clamp = DefaultIKGradientComponentClamp);
+                               const Properties& properties = Properties());
```

Comment:
Since we agreed on that the next release is major version up, we don't need to worry about the API compatibility here. Once we have strong reason for minor version up release in the future, we can consider back-port for this.